package.json

```diff
@@ -27,7 +27,8 @@
     "dependencies": [
       "ui5-middleware-livereload",
       "ui5-tooling-transpile",
-      "ui5-tooling-modules"
+      "ui5-tooling-modules",
+      "firebase"
```

Comment:
shouldn't be necessary - the ui5-tooling-modules use the native `require.resolve` lookup

Reply:
thought so